index.html

```diff
@@ -0,0 +1,198 @@
+            endgame.
+          </p>
+        </article>
+        <article class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fas fa-users"></i>
+          </div>
+          <h3 class="program-title">Connect</h3>
+          <p class="p-1">
+            Become part of the amazing snooker community to discuss, study and
+            play.
+          </p>
+        </article>
+      </section>
+      <button type="button" id="join-btn">Join Event</button>
+    </section>
+    <section class="feature-container d-flex flex-column align-items-center p-3 bg-white" id="player-section">
+      <h2 class="section-heading">Featured Players</h2>
+      <hr />
+      <section class="row justify-content-center">
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player1">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player2">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player3">
+   
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player4">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player5">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player6">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player7">
+          
+        </section>
+        <section class="col-md-5 d-flex align-items-center p-1 speaker" id="player8">
+          
+        </section>
+      </section>
+    </section>
+    <section class="partner-container1 d-flex flex-column align-items-center p-2" id="partner-section">
+      <h2 class="section-heading">Partners</h2>
+      <hr />
+      <ul class="partner-grid d-flex flex-wrap align-items-center p-3 gap-4 justify-content-center">
+        <li><img src="img/partner1.png" alt="algorand logo" /></li>
+        <li><img src="img/partner2.png" alt="algorand logo" /></li>
+        <li><img src="img/partner3.png" alt="algorand logo" /></li>
+        <li><img src="img/partner4.png" alt="algorand logo" /></li>
+        <li><img src="img/partner5.png" alt="algorand logo" /></li>
+      </ul>
+    </section>
+  </main>
+  <section class="py-3 mx-2 border border-3 text-center" id="more">
+    <strong>MORE</strong> <i class="fa-solid fa-chevron-down mx-2 arrow-down"></i>
+  </section>
+  <footer
+    class="d-flex justify-content-center flex-wrap align-items-center p-2 mt-3 mb-3 gap-3 bg-white footer-container" id="footer">
+    <img width="50px" height="60px" src="img/main-icon.png" alt="icon">
+    <h2>
+      World Snooker Championship
+    </h2>
+    <p class="text-center">
+      2022 &copy; World Snooker Championship. All Rights Reserved.
+    </p> 
+  </footer>
```

Comment:
- [ ]  Good job fixing the footer for the desktop version. As the previous reviewer mentioned, The footer contains a logo aligned to the left and text aligned to the right for the mobile version too.
## Figma design
![figma](https://user-images.githubusercontent.com/69077061/204098532-10554217-9a09-4034-abea-124d14d5142f.PNG)

## Your design
![Capture2](https://user-images.githubusercontent.com/69077061/204098394-296fedb7-958a-4235-a63e-1d70f2299fa4.PNG)